packages/genesys-spark-tokens/data/$themes.json

```diff
@@ -21,21 +21,6 @@
     "$figmaCollectionId": "VariableCollectionId:28327:137965",
     "$figmaModeId": "28327:0",
     "$figmaVariableReferences": {
-      "spacing.4xs": "56893a6774799a9563967ac995e533e01d87347e",
```

Comment:
@VirtualClover are all these deletions expected?

Reply:
Seems that the plugin deleted some references to figma styles, they should be fixed now 🫡